src/VoyagerServiceProvider.php

```diff
@@ -237,6 +251,24 @@ public function registerConfigs()
         );
     }
 
+    public function registerGates()
+    {
+        $dataTypes = VoyagerFacade::model('DataType')::get();
+
+        foreach ($dataTypes as $dataType) {
+            $policyClass = isset($this->policies[$dataType->model_name]) ? $this->policies[$dataType->model_name] : BasePolicy::class;
```

Comment:
@cotrudel, I'd leave this as BasePolicy all the time, and set the "PolicyName" field in the BREAD for those two through the seeder.